packages/world/src/World.sol

```diff
@@ -43,45 +44,47 @@ contract World is Store, IWorldCore, IErrors {
   /**
    * Install the given module at the given namespace in the World.
    */
-  function installModule(IModule module, bytes16 namespace) public {
-    // Prevent the same module from being installed multiple times in the same namespace
-    if (InstalledModules.get(ROOT_NAMESPACE, module.getName()).moduleAddress != address(0)) {
-      revert ModuleAlreadyInstalled(ResourceSelector.from(namespace, module.getName()).toString());
+  function installModule(IModule module, bytes memory args) public {
+    // Prevent the same module from being installed multiple times
+    // TODO: do we need this?
+    if (InstalledModules.get(module.getName(), keccak256(args)).moduleAddress != address(0)) {
+      revert ModuleAlreadyInstalled(ResourceSelector.from(module.getName()).toString());
     }
 
     Call.withSender({
       msgSender: msg.sender,
       target: address(module),
-      funcSelectorAndArgs: abi.encodeWithSelector(IModule.install.selector, namespace),
+      funcSelectorAndArgs: abi.encodeWithSelector(IModule.install.selector, args),
```

Comment:
Restricting the arguments passed to a module's install method to `namespace` was too restrictive. None of our current modules even need the `namespace` param, but some (like `IndexModule`) need some configuration to be passed to the `install` method.